core/domain/subscription_services_test.py

```diff
@@ -361,13 +391,33 @@ def setUp(self):
         self.owner_2_id = self.get_user_id_from_email(self.OWNER_2_EMAIL)
 
     def _get_all_subscribers_of_creators(self, user_id):
+        """Returns all the ids of the subscribers that have subscribed to the
+        creators.
```

Comment:
I think the creator should be singular (in the function name too). It seems to be related to the user_id. (Correct me if I am wrong here).

Reply:
Done. Changed function name too at all the relevant places.